### PR TITLE
Fix reek issue for duplicate methods issue #56

### PR DESCRIPTION
**Pull request by sweetstyle11**

### CODE REVIEW

File: app/views/connections/new.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 <% if session["errors"] != nil %>
   <% session["errors"].each do |message| %>
+  <p style="color: red">WARNING! <%= message%></p>
```
Comment (sumeetjain) [inline app/views/connections/new.erb]: Is this related to the refactoring? It doesn't look like it, but maybe I'm mistaken? Remember to keep each branch's footprint minimal.

**Comment by sweetstyle11** [comment]
Thanks Sumeet, you're right, that change that you highlighted was not directly related to refactoring. I caught that bug while I was refactoring our error methods, and I went ahead and fixed it on this branch without thinking about whether or not to cut a new branch to fix that bug. Just to understand proper procedure here, should I have cut a new branch specifically for fixing that one bug even though it was only adding one line of code?

**Comment by sumeetjain** [comment]
> should I have cut a new branch specifically for fixing that one bug even though it was only adding one line of code

Yup :)
